index.js

```diff
@@ -31,14 +32,20 @@ class Client {
     this.userAgent = options.userAgent || null
     this.middleware = []
 
+    if (options.tracer) {
+      this.annotateTrace = tracing({
+        tracer: options.tracer
+      })
+    }
+
     this.logMiddleware = this.logMiddleware.bind(this)
     this.callMiddleware = this.callMiddleware.bind(this)
 
     this.rebuildMiddleware()
   }
 
   makeHTTPRequest (body, options, fn) {
-    const requestOptions = {
+    var requestOptions = {
```

Comment:
we should default to `let` over `var`

Reply:
Correction: it should always be `const` first, then `let` and never `var`.